app/services/session_encryptor.rb

```diff
@@ -8,15 +8,13 @@ def self.build_user_access_key
     build_user_access_key
   end
 
-  # rubocop:disable Security/MarshalLoad
   def self.load(value)
     decrypted = encryptor.decrypt(value, user_access_key)
-    Marshal.load(::Base64.decode64(decrypted))
+    JSON.parse(decrypted, quirks_mode: true).with_indifferent_access
```

Comment:
It's for parsing single values (http://stackoverflow.com/a/17307387). I added it because it's enabled by default in the JSON serializer defined in the [redis-session-store](https://github.com/roidrage/redis-session-store/blob/master/lib/redis-session-store.rb#L166-L174) gem, as well as in the Rails custom JSON serializer: https://github.com/rails/rails/blob/2e421ad3df9918261a81f8dd602552685882f75e/activesupport/lib/active_support/json/encoding.rb#L99
I'm assuming they had good reasons for adding that option.

Speaking of which, I wonder if we should use `ActiveSupport::JSON` here or just straight up `JSON`. Rails seems to be doing some extra stuff, namely:

>  Rails does more escaping than the JSON gem natively does (we
     escape \u2028 and \u2029 and optionally >, <, & to work around
     certain browser problems).

Not sure if that affects us or not.